Med

sludge9999 opened the pull request:
Check

nkk71 committed:
Add minimal twrp for 9.0

Co-authored-by: nkk71
Co-authored-by: CaptainThrowback <captainthrowback@hotmail.com>
Co-authored-by: Chippa-a
Co-authored-by: imasaru
Co-authored-by: bigbiff bigbiff@teamw.in
Co-authored-by: mauronofrio <poronofrio@gmail.com>
Co-authored-by: theimpulson <aayushgupta219@gmail.com>

mauronofrio committed:
Move to TeamWin's android_system_update_engine

golden-guy committed:
manifest: Use 9.0.0_r49 repos for libcups and libhevc

Needed for ASB 2019-09

Change-Id: I12824c6c224c4c1ec8876c3717793a7795d246eb

theimpulson committed:
manifest: Stop tracking more apps

*None of them are required to build recovery

Signed-off-by: Aayush Gupta <aayushgupta219@gmail.com>

CaptainThrowback committed:
twrp: source repo from TeamWin

theimpulson committed:
manifest: Don't track generic device repo and kernel configs

*We supply our own device trees and kernel to build TWRP
*Not needed in compilation as well

MartinX3 committed:
Readd platform/system/libufdt to ensure dtbo building

Fixed https://github.com/minimal-manifest-twrp/platform_manifest_twrp_omni/issues/54

CaptainThrowback committed:
nos: add in host repos

Change-Id: I45d97fa97a4b4b7248cc557e70fef75c359eac04

CaptainThrowback committed:
Update branch on magisk-prebuilt repo

heybary420 commented:
README.md
omni-aosp.xml
omni-default.xml
remove.xml

heybary420 reviewed (changes requested):
principles

heybary420 commented on README.md:
```diff
@@ -34,7 +22,7 @@ after you type in `git commit -a` , type in the commit message and save. You wou
 git commit --amend --author "Author <email@address.com>"
```
https://gerrit.omnirom.org

heybary420 reviewed (commented):
cd rom 
git add -A
git commit -m git rebase -i HEAD
git push ssh://heybary420@gerrit.omnirom.org:29418/<project> HEAD:refs/for/android-9.0

heybary420 commented on README.md:
```diff
@@ -1,29 +1,17 @@
 ----------------------
 Maintaining authorship is a very important aspect of working with Open Source code. If you wish to submit a patch/fix
 from anywhere else (another ROM, project, etc.), it is imperative that you maintain the ownership of the person whose
-work you are seeking to include. Doing so will ensure that credit is given where it is deserved, and the [prinicples of open source](http://opensource.org/docs/osd)
```
http://opensource.org/docs/osd

heybary420 commented on README.md:
```diff
@@ -1,29 +1,17 @@
 ## Maintaining Authorship ##
 ----------------------
 Maintaining authorship is a very important aspect of working with Open Source code. If you wish to submit a patch/fix
 from anywhere else (another ROM, project, etc.), it is imperative that you maintain the ownership of the person whose
```
http://opensource.org/docs/osd

MartinX3 committed:
Update README.md

CaptainThrowback committed:
Use TeamWin android_build repo

Change-Id: Iafe1bec853b6f4db4f44e66d31a189ccdb581289

MartinX3 committed:
update README.md

switch from git:// to https://

fix https://github.com/minimal-manifest-twrp/platform_manifest_twrp_omni/issues/96

CaptainThrowback committed:
Add python3 repo

rumplestilzken committed:
Update to include python2 warning to the README.md

See #99 for discussion surrounding python2 warning to help developers navigate this particular change in infrastructure.

MartinX3 committed:
Merge pull request #100 from rumplestilzken/patch-1

Update to include python2 warning to the README.md
